### PR TITLE
Period Numbers Row

### LINKED ISSUE #121: Schedule do not a label for period order
**Is your feature request related to a problem? Please describe.**
According to the feedback from the teacher's UI test, all of them requested that it is possible to miss adding the schedule item due to the fact that there is no label for the period order causing them to manually count the period to get to where they need to add their schedule item. 

**Describe the solution you’d like**
There's another row on the top of the schedule similar to the physical one school gave out to students and teachers showing the period order and possibly the time of class as well.

### PR DESCRIPTION
**Pull request by SiravitPhokeed**
image

**Features**
- Period Numbers Row in Schedule
  Resolves #121
- Schedule uses more space when possible
- Renamed “Enter edit mode” to “Edit schedule”
- Example for room code

**Fixes**
- Welcome immediately causes 500 if the user is not logged in

### CODE REVIEW

File: src/utils/backend/classroom/classroom.ts [hunk 1]
```diff
@@ -409,4 +409,3 @@ export async function getClassStudentList(
     error: null,
   };
 }
-
```
Comment (Jimmy-Tempest) [inline src/utils/backend/classroom/classroom.ts:412]: trailing whitespace empty line

File: src/utils/backend/contact/index.ts [hunk 2]
```diff
@@ -80,4 +80,3 @@ export async function deleteContact(
   }
   return { data: deletedContact!, error: null };
 }
-
```
Comment (Jimmy-Tempest) [inline src/utils/backend/contact/index.ts:83]: this too

File: src/utils/backend/person/person.ts [hunk 3]
```diff
@@ -298,4 +298,3 @@ export async function getPersonIDFromUser(
 
   return { data: null, error: { message: "invalid role." } };
 }
-
```
Comment (Jimmy-Tempest) [inline src/utils/backend/person/person.ts:301]: This too

File: src/utils/backend/person/teacher.ts [hunk 4]
```diff
@@ -326,4 +326,3 @@ export async function getTeacherFromPublicUser(
 
   return await db2Teacher(supabase, data!);
 }
-
```
Comment (Jimmy-Tempest) [inline src/utils/backend/person/teacher.ts:329]: This too

File: src/utils/backend/statistic.ts [hunk 5]
```diff
@@ -39,4 +39,3 @@ export async function getOnboardStatistic(supabase: DatabaseClient) {
   };
   return statistics;
 }
-
```
Comment (Jimmy-Tempest) [inline src/utils/backend/statistic.ts:42]: This too

**Review by Jimmy-Tempest — APPROVED** [review]
LGTM! however, I don't know if you have your formatter configured correctly or if it's another convention I don't know about

File: src/utils/backend/classroom/classroom.ts (hunk 1 above)
Comment (SiravitPhokeed) [reply src/utils/backend/classroom/classroom.ts:412]: You had 2 line breaks here. Convention (and literally every other file in this project) says there should be one. I just corrected them.

**Pull request merged**